src/routes/blog/post/announcing-appwrite-integration-catalog/+page.markdoc

```diff
@@ -0,0 +1,132 @@
+---
+layout: post
+title: "Announcing Appwrite's new Integrations Catalog"
+description: We are beyond excited to announce the launch of our brand-new Appwrite Integrations Catalog. 
+date: 2024-07-29
+cover: /images/blog/integrations-catalog.png
+timeToRead: 12
+author: aditya-oberai
+category: product
+featured: true
+---
+
+This catalog is designed to empower you by providing a seamless way to integrate your favorite tools, libraries, and services with Appwrite, making your development process even more efficient and enjoyable.
```

Comment:
Would it be better to start this with `Our new Integrations catalog` instead of `This catalog`?